test/models/work_test.rb

```diff
@@ -0,0 +1,112 @@
+require "test_helper"
+
+describe Work do
+  let(:work) { works(:album1) }
+
+  describe 'relations' do
+      it 'has votes' do
+        work.must_respond_to :votes
+        expect(work.votes.first).must_be_kind_of Vote
+      end
+    end
+
+  	describe 'validations' do
+
+  		it 'is valid when title is present and unique to category' do
+  			#act
+  			is_valid = work.valid?
+
+  			#assert
+  			expect(is_valid).must_equal true
+  		end
+
+  		it 'is invalid without a title' do
+  			work.title = nil
+
+  			is_valid = work.valid?
+
+  			expect(is_valid).must_equal false
+  			expect(work.errors.messages).must_include :title
+  		end
+
+    it "will save if title is unique to category, but not overall" do
+      work2 = Work.new(title: 'white album', category: 'book')
+
+      is_valid = work2.valid?
+
+      expect(is_valid).must_equal true
+    end
+
+  	it "won't save if title is a duplicate within category" do
+  		work2 = Work.new(title: 'white album', category: 'album')
+
+  		is_valid = work2.valid?
+
+  		expect(is_valid).must_equal false
+
+  	end
+
+
+  end
+
+  describe 'count votes (inherited)' do
+
+      it 'accurately counts work votes' do
+        low = works(:album1)
+        high = works(:movie2)
+
+        expect(low.count_votes).must_equal 1
+        expect(high.count_votes).must_equal 2
+
+      end
+    end
+
+
+  describe 'categorize works' do
+
+      it 'returns only works from certain categories' do
+        album_array = Work.categorize_works('album')
+        expect(album_array.first.category).must_equal 'album'
+
+        book_array = Work.categorize_works('book')
+        expect(book_array.first.category).must_equal 'book'
+
+        movie_array = Work.categorize_works('movie')
+        expect(movie_array.first.category).must_equal 'movie'
+      end
+    end
+
+    describe 'sort works' do
+
+        it 'sorts works by most to least votes' do
```

Comment:
There's a lot of interesting test cases for your custom `Work` methods missing here. For `top_work`, I would wonder:

- What happens if there are no works?
- What happens if there are works but no votes?
- What happens if two works have the same number of votes?

Similarly for your category and top ten methods, I would ask:

- What if there are no works of that category?
- What if there are less than 10 works?
- What if there's a tie for last place, e.g. works 9, 10 and 11 all have 0 votes?